.github/workflows/android-release.yml

```
@@ -0,0 +1,58 @@
+          ESCAPED_PATHS="${FIND_COMMAND//$'\n'/%0A}"
+          
+          echo "APK_FILES=$ESCAPED_PATHS" >> $GITHUB_OUTPUT
+          
+      - name: 📝 Show built APK paths
+        run: |
+          echo "Found the following APKs to upload:"
+          echo "${{ steps.apk_paths.outputs.APK_FILES }}"
+
+      # Upload ALL the built APKs as release assets
+      - name: 📤 Upload ALL APKs to Release Assets
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)